pom.xml

```diff
@@ -1,272 +1,305 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xmlns="http://maven.apache.org/POM/4.0.0"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
-  <modelVersion>4.0.0</modelVersion>
+    <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.bynder</groupId>
-  <artifactId>bynder-java-sdk</artifactId>
-  <version>2.2.7</version>
-  <packaging>jar</packaging>
+    <groupId>io.github.arpit-sharma-usc</groupId>
+    <artifactId>byndertest-java-sdk</artifactId>
+    <version>2.2.9</version>
```

Comment:
I don't think we need a version change for this PR?

Reply:
No please ignore this as I said I add this to test it with my test-account. (That already has 2.2.8) I original version is still 2.2.7 in the [PR](https://github.com/Bynder/bynder-java-sdk/pull/43/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R10)